frontend/packages/console-dynamic-plugin-sdk/scripts/generate-doc.ts

```diff
@@ -76,6 +76,9 @@ const generateDoc = (comment: tsdoc.DocComment) => {
   const summary = renderDocNode(comment.summarySection);
   const exampleBlock = comment.customBlocks.find((block) => block?.blockTag?.tagName === EXAMPLE);
   const example = renderDocNode(exampleBlock?.content);
+  // FIXME: Let tsdoc parser do the work instead of trying to manually parse the name and content.
```

Comment:
@TheRealJon I wonder if we could tackle this as part of https://bugzilla.redhat.com/show_bug.cgi?id=2083154?

Reply:
Definitely. I've already started on this 😄